src/thunderbots/software/param_server/cfg/Params.cfg

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import roslib
+PACKAGE_NAME = "param_server"
```

Comment:
Just following up from the comments in the previous round of review, would be helpful to add a comment explaining what this `PACKAGE_NAME` is actually for, and maybe changing it to something like `PARAMS_CONFIG_NAMESPACE`?